core/commons/che-core-commons-inject/src/main/java/org/eclipse/che/inject/lifecycle/DestroyErrorHandler.java

```diff
@@ -21,12 +23,8 @@
     void onError(Object instance, Method method, Throwable error);
 
     /**
-     * Implementation of DestroyErrorHandler that ignore errors, e.g. such behaviour is required for annotation {@link
+     * Implementation of DestroyErrorHandler that log errors, e.g. such behaviour is required for annotation {@link
```

Comment:
actually `@javax.annotation.PreDestroy` does require the logging of errors and provided documentation can confused a bit.